### PR TITLE
Make output of `render_to_string` fixed

### LINKED ISSUE #320: Stable render_to_string
When I add some test contains `sycamore::generic_node::ssr_node::render_to_string`, these tests randomly fails.

I found this is caused by `AHashMap`, if there is a stable output then it will be very useful for testing.

This is not used in a production environment, only for testing.

### PR DESCRIPTION
**Pull request by oovm**
Fix #320

### CODE REVIEW

File: packages/sycamore/src/generic_node/ssr_node.rs [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-
-use ahash::AHashMap;
-use once_cell::sync::Lazy;
 use wasm_bindgen::prelude::*;
 
 use crate::generic_node::{GenericNode, Html};
 use crate::reactive::create_scope;
 use crate::utils::hydrate::{get_next_id, with_hydration_context};
 use crate::view::View;
+use indexmap::map::IndexMap;
```
Comment (lukechu10) [inline packages/sycamore/src/generic_node/ssr_node.rs:15]: Nit: Can you organize the imports here please? The order should be std, dependencies, self.

**Review by lukechu10 — COMMENTED** [review]
lgtm

File: packages/sycamore/src/generic_node/ssr_node.rs (hunk 1 above)
Comment (oovm) [reply packages/sycamore/src/generic_node/ssr_node.rs]: Sorry I didn't notice this, IDEA auto import breaks this part

**Comment by lukechu10** [comment]
Seems like the build website workflow is broken. I guess I'll need to fix that.

**Pull request merged**